go/inst/analysis.go

```diff
@@ -64,6 +64,7 @@ const (
 	DifferentGTIDModesStructureWarning                                       = "DifferentGTIDModesStructureWarning"
 	ErrantGTIDStructureWarning                                               = "ErrantGTIDStructureWarning"
 	NoFailoverSupportStructureWarning                                        = "NoFailoverSupportStructureWarning"
+	NoWriteableNodesWarning                                                  = "NoWriteableNodesStructureWarning"
```

Comment:
We should be concerned with non-writable master, for sure. The rest of the nodes are practically irrelevant (that is, we expect them to be read-only anyhow).

Reply:
I'd name this: `NonWritableMasterWarning`

Reply:
@shlomi-noach Make sense, I've updated to warn if the current master is `read-only`.